### PR TITLE
FRW-9407 Fixed robot tests for missing DataExchangeAPI complex request fields.

### PR DESCRIPTION
**Pull request by asmarovydlo**
https://spryker.atlassian.net/browse/FRW-9407

BC-check https://github.com/spryker/suite-nonsplit/pull/9189

### CODE REVIEW

File: tests/api/suite/bapi/dynamic_entity/complex/negative.robot [hunk 1]
```diff
@@ -273,11 +273,11 @@ Upsert_product_abstract_collection_with_missing_required_field:
     When Save value to a variable:    [data][0][robotTestsProductAbstractProducts][0][id_product]    id_product
     ### UPDATE PRODUCT ABSTRACT WITH INVALID CHILD CONTAINED INVALID FIELD ###
     And I set Headers:    Content-Type=application/json    Authorization=Bearer ${token}
-    And I send a PUT request:   /dynamic-entity/robot-tests-product-abstracts  {"data": [{"id_product_abstract": ${id_product_abstract}, "fk_tax_set": 1, "attributes": "FOO", "sku": "FOO", "robotTestsProductAbstractProducts": [{"fk_product_abstract": ${id_product_abstract}, "attributes": "FOOBAR", "sku": "FOOBAR"}]}]}
+    And I send a PUT request:   /dynamic-entity/robot-tests-product-abstracts?XDEBUG_SESSION_START=PHPSTORM  {"data": [{"id_product_abstract": ${id_product_abstract}, "fk_tax_set": 1, "attributes": "FOO", "sku": "FOO", "robotTestsProductAbstractProducts": [{"fk_product_abstract": ${id_product_abstract}, "attributes": "FOOBAR"}]}]}
```
Comment (aleksandr-velikanov) [inline tests/api/suite/bapi/dynamic_entity/complex/negative.robot:276]: XDEBUG_SESSION_START -- why is this needed?

**Comment by annakotiuk** [comment]
Is there a shop running with fixed robot tests?
